central/notifiers/generic/generic.go

```diff
@@ -179,7 +179,7 @@ func (g *generic) Test(ctx context.Context) *notifiers.NotifierError {
 }
 
 func (g *generic) constructJSON(message protocompat.Message, msgKey string) (io.Reader, error) {
-	msgStr := protojson.Format(message)
+	msgStr := protojson.MarshalOptions{}.Format(message)
```

Comment:
We have a few wrappers. Can you please take a look to see if that would work?

i.e.
```
	msgStr, err := jsonutil.MarshalToCompactString(message)
	if err != nil {
		return nil, err
	}
```
maybe some other function from `jsonutil` pkg would work.